python/Ganga/Runtime/Repository_runtime.py

```diff
@@ -134,7 +134,15 @@ def bootstrap():
         registry.startup()
         logger.debug("started " + registry.info(full=False))
         if registry.name == "prep":
-            registry.print_other_sessions()
+            # Get a list of strings describing other running sessions
+            other_sessions = registry.get_other_sessions()
```

Comment:
And I would argue that the *registry* should not have to care about other *repository* sessions. We do care about the internals of the registry (in as much as the repository is an internal of the registry) since it is as the repository level where this is dangerous.

Maybe this warning would make a lot more sense being in `PrepRegistry.startup()`? Especially given that `if registry.name == "prep"` code-smell above.